mono/metadata/threadpool.c

```diff
@@ -48,6 +48,9 @@
 #include <mono/utils/refcount.h>
 #include <mono/utils/mono-os-wait.h>
 
+// consistency with coreclr https://github.com/dotnet/coreclr/blob/master/src/vm/win32threadpool.h#L111
```

Comment:
please don't use a link to `master` as that'll quickly get outdated. always use a permalink like https://github.com/dotnet/coreclr/blob/643b09f966e68e06d5f0930755985a01a2a2b096/src/vm/win32threadpool.h#L111